File: search-rnd-array-element.js
function getRandom(max)
{
  return parseInt(Math.random() * max);
}
let n = 10;
var a = [];
var out = '';
for (i = 0; i < n; i++){
	a[i] = getRandom(15);
	out += a[i] + ' ';
}
console.log('Filled random array: ');
console.log(out);

(function (){
	out = '';
	var isFound = false;
	let k = prompt('Введите искомое число:', 0);
	if ((k == 0 || k ^ 0 == k) && k >= 0) ;
	else{
		alert('Вводимое число не является положительным целым числом!');
		return;
	}
	
	for (i = 0; i < n; i++){
		if (a[i] == k){
			isFound = true;
			out = 'Element ' + k + ' is found!';
			break;
		}
	}
	if (!isFound){
		out = 'Element ' + k + ' not found!';
	}
	console.log(out);
})();

File: search-rnd-array-element.pas
program SearchRndElementInRndArray;
const N = 10;
var
  a: array [1..N] of integer;
  k: integer;
  isFound: boolean;
  i: integer;
begin
  randomize;

  for i := 1 to N do
    a[i] := Random(15);
  
  writeln('Filled random array: ');
  for i := 1 to N do
    write(a[i], ' ');
  
  writeln;
 
  k := Random(15);
 
  IsFound := False;
  for i:=1 to N do
    if a[i] = k then 
    begin
      IsFound := True;
      break;
    end;
 
  if IsFound then
    writeln('Element ', k, ' is found!')
  else
    writeln('Element ', k, ' not found');
	
end.